# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/HttpTransactionChild.cpp
# Commit: 23b864e14db0
# Full Hash: 23b864e14db0cf874d0e126b6299c8c44922dde3
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2022-11-16 09:31:06
# Regressor Bug: 1791633
# File Overlap Count: 1
# Description:
#   Bug 1791633 - separate nsITLSSocketControl from nsITransportSecurityInfo r=necko-reviewers,kershaw,jschanck
#   
#   Depends on D160311
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D160313
# ==============================================================================

diff -r 0bcba3375ec0 -r 23b864e14db0 netwerk/protocol/http/HttpTransactionChild.cpp
--- a/netwerk/protocol/http/HttpTransactionChild.cpp	Tue Nov 15 20:03:29 2022 +0000
+++ b/netwerk/protocol/http/HttpTransactionChild.cpp	Tue Nov 15 20:03:29 2022 +0000
@@ -415,9 +415,6 @@
         !protocol.IsEmpty()) {
       mProtocolVersion.Assign(protocol);
     }
-    // Make sure peerId is generated.
-    nsAutoCString unused;
-    securityInfo->GetPeerId(unused);
   }
 
   UniquePtr<nsHttpResponseHead> head(mTransaction->TakeResponseHead());